### PR TITLE
separate best bid/ask channel subscribe

### PR DESCRIPTION
**Pull request by kilu01**

### CODE REVIEW

File: lib/ex_huobi/margin/websocket/market_ws.ex [hunk 1]
```diff
@@ -8,12 +8,14 @@ defmodule ExHuobi.Margin.WebSocket.MarketWs do
 
       def start_link(args \\ %{}) do
         subscription = args[:subscribe] || ["market.btcusdt.mbp.150"]
+        bbo_subscription = args[:bbo_subscribe] || ["market.btcusdt.bbo"]
```
Comment (sirius248) [inline lib/ex_huobi/margin/websocket/market_ws.ex:11]: @quantd2 I don't think we need to introduce this at all. `args[:subscribe]` can be a list of channel already. 

`args[:subscribe]` can pontentially a list of `["market.btcusdt.mbp.150", "market.btcusdt.bbo"]` -> then simply loop through each channel to subscribe.

File: lib/ex_huobi/margin/websocket/market_ws.ex [hunk 2]
```diff
@@ -117,7 +119,7 @@ defmodule ExHuobi.Margin.WebSocket.MarketWs do
 
       @impl true
       def handle_info(:subscribe_delta, state) do
-        topics = Map.get(state, :subscribe)
+        topics = Map.get(state, :subscribe) ++ Map.get(state, :bbo_subscribe)
```
Comment (sirius248) [inline lib/ex_huobi/margin/websocket/market_ws.ex:122]: then here is just the list of topic. And you already loop through. Feel like you don't even need to have this PR

File: lib/ex_huobi/margin/websocket/market_ws.ex (hunk 1 above)
Comment (kilu01) [reply lib/ex_huobi/margin/websocket/market_ws.ex:11]: the thing is for snapshot event, I need to send a subscribe request `{"req": topic}` which will cause error for best bid/ask topic

Comment (sirius248) [reply lib/ex_huobi/margin/websocket/market_ws.ex:11]: we can pattern match the specific event, and send the correct operation payload to that one right?

**Review by sirius248 — APPROVED** [review]

**Pull request merged**